BankSDK/GiniBankSDK/Sources/GiniBankSDK/Resources/de.lproj/Localizable.strings

```diff
@@ -86,22 +86,22 @@
 "ginibank.digitalinvoice.edit.price.error" = "Bitte geben Sie den Artikelpreis ein.";
 
 "ginibank.skonto.screen.title" = "Skonto";
-"ginibank.skonto.backButton.title" = "Zurück";
-"ginibank.skonto.info.title" = "Mit Skonto";
-"ginibank.skonto.info.status" = "• Aktiviert";
-"ginibank.skonto.info.message" = "Zahlung in %@: %@ Skonto.";
-"ginibank.skonto.info.amount.title" = "Betrag nach Abzug";
-"ginibank.skonto.info.date.title" = "Ablaufdatum Skonto";
-"ginibank.skonto.notapplied.title" = "Ohne Skonto";
-"ginibank.skonto.notapplied.amount.title" = "Betrag ohne Abzug";
-"ginibank.skonto.total.amount.skonto" = "%@ Skonto";
-"ginibank.skonto.paybutton.title" = "Zahlung fortsetzen";
+"ginibank.skonto.backbutton.title" = "Zurück";
+"ginibank.skonto.withdiscount.title" = "Mit Skonto";
+"ginibank.skonto.active" = "• Aktiviert";
+"ginibank.skonto.infobanner.normal.message" = "Zahlung in %@: %@ Skonto.";
```

Comment:
[Suggestion] @ValentinaIancu-Gini what do you think about this rename? infobanner.normal -> infobanner.default

Reply:
I think is a good idea. I will talk with Mahdi to see if they want to use the same version.